index.js

```diff
@@ -48,6 +37,7 @@ function getListLocales() {
         reject(e);
       });
     } else {
+      supportedLocales = supportedLocales.split(",").map(item => item.trim());
```

Comment:
in that case, why not just keep it documented that the list should be CSV, no spaces, and not bother to trim? Saves a bit of work.